src/sidebar/components/Annotation/test/AnnotationEditor-test.js

```diff
@@ -477,6 +477,148 @@ describe('AnnotationEditor', () => {
     });
   });
 
+  context('when annotation author is a third party user', () => {
+    function insertMentionSuggestion(wrapper, user) {
+      wrapper.find('MarkdownEditor').props().onInsertMentionSuggestion(user);
+    }
+
+    it('initializes display names map with annotation mentions', () => {
+      const mentions = [
+        {
+          userid: 'acct:ignored@example.com',
+        },
+        {
+          userid: 'acct:foo@example.com',
+          display_name: 'Foo',
+          username: 'foo',
+        },
+        {
+          userid: 'acct:bar@example.com',
+          display_name: 'Bar',
+          username: 'bar',
+        },
+      ];
+      const annotation = {
+        ...fixtures.defaultAnnotation(),
+        mentions,
+        user: 'acct:username@example.com', // Third party user
+      };
+      const wrapper = createComponent({ annotation });
+
+      wrapper.find('AnnotationPublishControl').props().onSave();
+
+      assert.calledWith(
+        fakeAnnotationsService.save,
+        annotation,
+        sinon.match({
+          mentionMode: 'display-name',
+          usersMap: new Map([
+            [
+              'Foo',
+              {
+                userid: 'acct:foo@example.com',
+                displayName: 'Foo',
+                username: 'foo',
+              },
+            ],
+            [
+              'Bar',
+              {
+                userid: 'acct:bar@example.com',
+                displayName: 'Bar',
+                username: 'bar',
+              },
+            ],
+          ]),
+        }),
+      );
+    });
+
+    it('tracks every inserted mention suggestion', () => {
```

Comment:
```suggestion
    it('tracks user info for inserted mention suggestions', () => {
```

Expanded to clarify what exactly is being tracked.